app/components/dropdowns/dropdown-single-select/component.tsx

```diff
@@ -0,0 +1,173 @@
+import React from 'react';
+import { useSelect } from 'downshift';
+import ARROW_DOWN_SVG from 'svgs/ui/arrow-down.svg';
```

Comment:
```suggestion
import ARROW_DOWN_SVG from 'svgs/ui/arrow-down.svg?sprite';
```

If you are not in a story from storybook, remember to put `?sprite`